package.json

```diff
@@ -41,7 +41,7 @@
     "lint:scss": "stylelint {demosplan/DemosPlanCoreBundle/Resources/client/scss,projects}/**/*.scss"
   },
   "dependencies": {
-    "@demos-europe/demosplan-ui": "^0",
+    "@demos-europe/demosplan-ui": "0.0.19",
```

Comment:
Das sollte eigentlich nicht nötig sein. Die Änderungen in der yarn.lock sollten eigentlich reichen.